README.md

```diff
@@ -1 +1,222 @@
+
+##### Input
+
+| **Argument Name** | **Type** | **Description** | **Required** |
+| --- | --- |--- | --- |
+| Threat Response API Client Id | Secure String | Threat Response API Client ID | Yes | 
+| Threat Response API Client Password | Secure String | Threat Response API Client Password | Yes | 
+| Cohesity Helios API Key | Secure String | API Key to access Helios | Yes | 
+| Cohesity Helios Ransomware Alerts Filter  | String | Number of hours, used to get anomalous objects detected in the last `N` hours| Yes | 
+| Create ServiceNow Incidents | String | Set this variable to `yes` or `no` based on which ServiceNow incidents will be created | No | 
+
+##### Output
+
+N/A
+
+### <a name="push-alerts-to-pi"></a> Push Helios Ransomware Alerts to Private Intelligence
+[top](#Cisco-SecureX-Integration)
+
+This workflow pushes Cohesity Helios ransomware alerts detected in the last `N` hours to Threat Response private intelligence data store. 
+
+
+This workflow can also be triggered based on a schedule. The schedule that is used here is `Cohesity Helios Ransomware Data Push Schedule` which is a schedule to trigger orchestration workflow to push Helios ransomware data to private intelligence.
+
+##### Input
+
+| **Argument Name** | **Type** | **Description** | **Required** |
+| --- | --- |--- | --- |
+| Threat Response API Client Id | Secure String | Threat Response API Client ID | Yes | 
+| Threat Response API Client Password | Secure String | Threat Response API Client Password | Yes | 
+| Cohesity Helios API Key | Secure String | API Key to access Helios | Yes | 
+| Cohesity Helios Ransomware Alerts Filter  | String | Number of hours, used to get anomalous objects detected in the last `N` hours| Yes | 
+
+##### Output
+
+N/A
+
+
+### <a name="ignore-anomaly-workflow"></a> Ignore Anomaly on Cohesity Helios
+[top](#Cisco-SecureX-Integration)
+
+This workflow ignores anomaly on for the specified object on Helios which removes this object from the list of anomalous objects. 
+
+> This is an Response workflows. 
+
+##### Input
+
+| **Argument Name** | **Type** | **Description** | **Required** |
+| --- | --- |--- | --- |
+| Threat Response API Client Id | Secure String | Threat Response API Client ID | Yes | 
+| Threat Response API Client Password | Secure String | Threat Response API Client Password | Yes | 
+| Cohesity Helios API Key | Secure String | API Key to access Helios | Yes | 
+| Cohesity Delete Sighting | String | Specifies where to delete or not the sighting once the anomaly is ignored. Can be `yes` or `no`. | Yes | 
+
+##### Output
+
+N/A
+
+### <a name="restore-anomaly"></a> Cohesity Restore Anomalous Object
+[top](#Cisco-SecureX-Integration)
+
+This workflow restores the specified anomalous object to the latest clean snapshot
```

Comment:
mention that this workflow also resolves the alert on helios once the restore task is triggered. and the restored vm name will in the format Recover-{original VM name}-VM-{epoch timestamp when the vm is restored} and the restore task name on the cluster will be in format Cisco_SecureX_triggered_restore_task_{object name}